src/main/webapp/WEB-INF/views/DesignPage.html

```diff
@@ -0,0 +1,339 @@
+                        cillum dolore eu fugiat nulla pariatur.</p>
+                    <hr>
+                    <div class="row">
+                        <div class="col-xs-6">
+                            <p>min<br><i class="fa fa-dollar"></i>100000 <br></p>
+                        </div>
+                        <div class="col-xs-6">
+                            <p>Still needed</br><i class="fa fa-dollar"></i>9999<br></p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="item">
+            <div class="thumbnail">
+                <div class="caption centered">
+                    <a href="#"><h3>Project name</h3></a>
+                </div>
+                <a href="#"><img src="http://placehold.it/600x340" alt="" class="img-responsive"></a>
+                <div class="caption">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                        ut
+                        aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                        cillum dolore eu fugiat nulla pariatur.</p>
+                    <hr>
+                    <div class="row">
+                        <div class="col-xs-6">
+                            <p>min<br><i class="fa fa-dollar"></i>100000 <br></p>
+                        </div>
+                        <div class="col-xs-6">
+                            <p>Still needed</br><i class="fa fa-dollar"></i>9999<br></p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="item">
+            <div class="thumbnail">
+                <div class="caption centered">
+                    <a href="#"><h3>Project name</h3></a>
+                </div>
+                <a href="#"><img src="http://placehold.it/600x340" alt="" class="img-responsive"></a>
+                <div class="caption">
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                        labore
+                        et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi
+                        ut
+                        aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+                        cillum dolore eu fugiat nulla pariatur.</p>
+                    <hr>
+                    <div class="row">
+                        <div class="col-xs-6">
+                            <p>min<br><i class="fa fa-dollar"></i>100000 <br></p>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)